fix(docs): update form screenshots as per latest web browser. (#27436)
mdn/content: files/en-us/learn/forms/basic_native_form_controls/index.md

Single line text fields have only one true constraint: if you type text with line breaks, the browser removes those line breaks before sending the data to the server.

> _The following screenshot shows default, focused and disabled text input types in Firefox 114 and Safari on macOS and in Chrome 114 and Edge 114 on Windows 11._

Reviewer: Drive-by comment/suggestion: Stating the browser and OS versions here creates the dependency to always maintain/update the text and screenshot going forward. How about we remove the version numbers?

```suggestion
_The following screenshot gives you an idea of how the text input field looks in its various states – by default, when it has focus, and when it is marked disabled – across Firefox and Safari on macOS and in Chrome and Edge on Windows._
```

I would even argue if adding this screenshot here is really helpful (why do we need to show both Chrome and Edge) and perhaps we should remove the screenshot and the accompanying text altogether. But if it has to be retained, I think we should make the text generic as suggested above to reduce maintenance.

Reviewer 2: @dipikabh Thanks very much. My take on this is that in a learn topic it is useful to explain what the controls look like and how they might differ. However given that all browsers do focus, disabled, default somewhat similarly I'm not sure that showing all of the different browsers is worth it. Further, 
- this is going to be an ongoing maintenance issue- will have to do this again in a number of releases.
- there are a bunch of other images further down in the doc.

How about we replace all the images with images from one browser, which we don't identify, and have words to describe what you're seeing. So "mocked up", something like:

----

The screenshot below shows a text input in default, focused, and disabled states. Most browsers indicate focus with a focus ring around the control and the disabled state using grey text or a faded/semi-opaque control.

![image](https://github.com/mdn/content/assets/5368500/6c210ead-01c1-4dd8-a795-300a3021b9a1)

----

And do something similar for the other controls.

Thoughts?

Author: So from discussion, what i understand is
- we need screenshot of default, focused and disabled state from only one browser (will prefer Chrome on macOS).
- screenshot and text changes need to be done for all the images present in doc.
These changes i can do.

Reviewer 2: Yes, except:

> screenshot and text changes need to be done for all the images present in doc.

All sections where you're showing off a feature with UI should have an image - even if there is not an image in the section currently (so not necessarily "update the current pics" - there might be more.

Yes, just one browser. Might be worth saying on the first use that these are from macOS but other browsers use similar highlighting techniques.

Single line text fields have only one true constraint: if you type text with line breaks, the browser removes those line breaks before sending the data to the server.

_The screenshot below shows a text input in default, focused, and disabled states. Most browsers indicate focus with a focus ring around the control and the disabled state using grey text or a faded/semi-opaque control._

![Screenshot of the default, focused and disabled states text input in Chrome on macOS](disabled.png)

> _Screenshots used in this document are taken in Chrome 115 on macOS. There could be little changes when these fields/buttons are displayed in different browser with different versions, but basic highlighting technique is similar._

Reviewer: ```suggestion
The screenshots used in this document were taken in the Chrome browser on macOS. There may be minor variations in these fields/buttons across different browsers, but the basic highlighting technique remains similar._
```

</fieldset>
```

> The following screenshot shows default, focused and disabled checkboxes. Default and disabled checkboxes are in checked state where as focused checkbox is in unchecked state with focus ring around it.

Reviewer: ```suggestion
The following screenshot shows checkboxes in the default, focused, and disabled states. Checkboxes in the default and disabled states appear checked, whereas in the focused state, the checkbox is unchecked, with focus ring around it.
```

</fieldset>
```

> The following screenshot shows default and disabled radio buttons in checked state along with focused radio button in unchecked state.

Reviewer: ```suggestion
The following screenshot shows default and disabled radio buttons in the checked state, along with focused a radio button in the unchecked state.
```

> ![Radio buttons in chrome 115 on macOS](radios.png)

Reviewer: ```suggestion
![Default, focused and disabled Radio buttons in chrome 115 on macOS](radios.png)
```

Buttons always behave the same whether you use a {{HTMLElement("button")}} element or an {{HTMLElement("input")}} element. As you can see from the examples, however, {{HTMLElement("button")}} elements let you use HTML in their content, which is inserted between the opening and closing `<button>` tags. {{HTMLElement("input")}} elements on the other hand are {{glossary("void element", "void elements")}}; their displayed content is inserted inside the `value` attribute, and therefore only accepts plain text as content.

> The following examples show default, focused, and disabled button input types.

Reviewer: ```suggestion
The following screenshot shows a button in the default, focused, and disabled states. In the focused state, there is a focus ring around the button, and in the disabled state, the button is greyed out.
```

> ![Default, focused and disabled button input types in chrome 115 on macOS](buttons.png)

Reviewer: ```suggestion
![Default, focus, and disabled button states in chrome 115 on macOS](buttons.png)
```

<input type="file" accept="audio/*;capture=microphone" />
```

> The following screenshot shows File picker widget in default, focused and disabled state when no file is selected.
> 
> ![File picker widget in chrome 115 on macOS](filepickers.png)

Reviewer: ```suggestion
The following screenshot shows the file picker widget in the default, focus, and disabled states when no file is selected.

![File picker widget in default, focus, and disabled states in chrome 115 on macOS](filepickers.png)
```
